sdk/eventgrid/azure-eventgrid/migration_guide.md

```diff
@@ -0,0 +1,84 @@
+# Guide for migrating to azure-eventgrid v2.0 from azure-eventgrid v1.3
+
+This guide is intended to assist in the migration to azure-eventgrid v2.0 from azure-eventgrid v1.3. It will focus on side-by-side comparisons for similar operations between the two packages.
+
+Familiarity with the azure-eventgrid v1.3 package is assumed. For those new to the eventgrid client library for Python, please refer to the [README for azure-eventgrid v2.0](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/eventgrid/azure-eventgrid/README.md) rather than this guide.
+
```

Comment:
Missing section `## Table of contents`.

check key-vault PR: https://github.com/Azure/azure-sdk-for-python/pull/15819/files